### ISSUE TITLE
Search Field example is using incorrect data type

### ISSUE THREAD

**Issue by StephenEsser** [cerner/terra-core#725]
# Issue Description

The Search Field example template is using a string for the minimum search text length prop which causes a warning in development. 

```
warning.js:35 Warning: Failed prop type: Invalid prop `minimumSearchTextLength` of type `string` supplied to `SearchField`, expected `number`.
    in SearchField (created by SearchFieldExampleTemplate)
    in SearchFieldExampleTemplate (created by SearchFieldExample)
    in SearchFieldExample (created by SearchFieldExamples)
    in div (created by SearchFieldExamples)
    in SearchFieldExamples (created by RouterContext)
    in div (created by Base)
    in div (created by I18nProvider)
    in IntlProvider (created by I18nProvider)
    in I18nProvider (created by Base)
    in Base (created by App)
    in div (created by SlidePanel)
    in div (created by SlidePanel)
    in SlidePanel (created by App)
    in div (created by ContentContainer)
    in div (created by ContentContainer)
    in div (created by ContentContainer)
    in ContentContainer (created by App)
    in App (created by RouterContext)
    in RouterContext (created by Router)
    in Router
``` 
## Issue Type

- [ ] New Feature
- [ ] Enhancement
- [X] Bug
- [ ] Other

## Expected Behavior

The example should use a number

## Current Behavior

The example uses a string

## Steps to Reproduce

1. Navigate to a current PR heroku deployment or run terra-site locally in development
2. Click on the Search Field from the side navigation
3. Observe the console warning

The code can be found here: https://github.com/cerner/terra-core/blob/master/packages/terra-site/src/examples/search-field/SearchFieldExampleTemplate.jsx#L13

## Environment

* Component Version:
* Browser Name and Version:
* Operating System and version (desktop or mobile):

**Comment by emilyrohrbough** [comment]
Issue found here: https://github.com/cerner/terra-core/blob/master/packages/terra-site/src/examples/search-field/SearchFieldExampleTemplate.jsx#L13

The default value is a string instead of a number.

**Comment by bjankord** [comment]
Resolved is #763
